product_order_noname/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Vauxoo, S.A. de C.V.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Product Order No Name',
+    'summary': 'Speedup product retrieve',
+    "version": "12.0.1.0.0",
+    "development_status": "Production/Stable",
+    "category": "Uncategorized",
+    "website": "https://github.com/OCA/<repo>" or "https://github.com/OCA/<repo>/tree/<branch>/<addon>",
+    "author": "Vauxoo, S.A. de C.V.",
+    "maintainers": ["vauxoo"],
```

Comment:
Check the key comment here:
 - https://github.com/OCA/maintainer-tools/blob/f1b559574648c01558b543e7550f5b9be7c92784/template/module/__manifest__.py#L13

 - `"your-github-login"`

you can use 

```suggestion
    "maintainers": ["WR-96", "moylop260"],
```